### DISCUSSION TITLE
Configure Python Version of ldap2pg v5.9

### DISCUSSION THREAD

**Discussion by peteerwhite** [dalibo/ldap2pg#637] [Q&A]
Hello everyone,

is it possible to tell ldap2pg v5.9 which Python version to use ? 

Kind regards and thanks for help :)

**Comment by bersace** [comment]
Hi. Not from packages. Use pip3.X to install ldap2pg on another Python version.

**Comment by peteerwhite** [comment]
Okay thanks. 

I am now trying to switch to ldap2pg v.6.

I am wondering if it is possible to start ldap2pg without specifying an LDAPUSER and LDAPPASSWORD

13:19:28 ERROR  Fatal error.                                     err="LDAP Result Code 206 \"Empty password not allowed by the client\": ldap: empty password not allowed by the client"

**Comment by bersace** [comment]
Hi @peteerwhite , Ho do you configure ldapsearch ?

**Comment by peteerwhite** [comment]
ldapsearch -x -H ldapuri -b ou=...,o=...,c=de cn=... member

**Comment by bersace** [comment]
What is the output of ldapsearch with `-d 5` option ?

**Comment by peteerwhite** [comment]
For what specify part are you looking for ?

**Comment by peteerwhite** [comment]
Is there a way to disable LDAP Authentification ?

**Comment by bersace** [comment]
> Is there a way to disable LDAP Authentification ?

LDAP supports anonymous binding. ldap2pg does not support this. I suggest you to open a new issue for this feature.
